api/Template/ListTemplates.php

```diff
@@ -3,30 +3,25 @@
 
 namespace HomeCEU\DTS\Api\Template;
 
-use Exception;
 use HomeCEU\DTS\Api\ResponseHelper;
 use HomeCEU\DTS\Entity\Template;
-use HomeCEU\DTS\UseCase\ListTemplates as ListTemplatesUseCase;
 use HomeCEU\DTS\Persistence\CompiledTemplatePersistence;
 use HomeCEU\DTS\Persistence\TemplatePersistence;
 use HomeCEU\DTS\Repository\TemplateRepository;
-use Slim\Http\Response;
+use HomeCEU\DTS\UseCase\ListTemplates as ListTemplatesUseCase;
 use Psr\Container\ContainerInterface;
 use Psr\Http\Message\ServerRequestInterface as Request;
+use Slim\Http\Response;
```

Comment:
I use a key command to remove unused `use` statements, PhpStorm apparently organizes use calls when you do this.